esp/scm/ws_dfu.ecm

```diff
@@ -707,6 +707,28 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] DFUGetFileMetaDataRe
 };
 
 
+ESPrequest ListHistoryRequest
+{
+    string name;
+};
+
+ESPresponse [exceptions_inline, nil_remove] ListHistoryResponse
+{
+    binary xmlmap;
+};
+
+ESPrequest EraseHistoryRequest
+{
+    string name;
+};
+
+ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
+{
+    binary xmlmap;
```

Comment:
@afishbeck - curious whether binary is best format for response? This is xml coming back, so needs serializing/deserializing as it stands.